src/uninstall.js

```diff
@@ -1,9 +1,15 @@
 const fs = require('fs');
 
+module.exports = function (api) {
 // get the graphql html that will be removed from 'src/index.template.html'
-const graphqlHtml = require('./exports').graphqlHtml;
+  let graphqlHtml
+
+  if (api.getPackageVersion('@quasar/app') >= '2.0.0') {
```

Comment:
This is a possible point of failure. Instead, use the `semver` package. Then you can do something like this:
```js
if (semver.gte(api.getPackageVersion('@quasar/app'), '2.0.0')) {
```

Reply:
Thanks @hawkeye64 

I checked the [Uninstall API](https://quasar.dev/app-extensions/development-guide/uninstall-api) and it does not have the function `getPackageVersion`, so for the moment I added a commit to revert the changes to `uninstall.js`.

We will need a way to get the package version of `@quasar/app`, or drop support for versions less than `2.0.0`

Reply:
@ejez - Can you make that change please?

Scott

Reply:
I have done the same as you guys are trying with both qenv and dotenv app-extensions

```js
  const version = api.getPackageVersion('@quasar/app')
  const v1 = semver.lt(version, '2.0.0')
```

If you are missing the api, then you need to do something like this:
```js
module.exports = function (api) {
  // Quasar compatibility check; you may need
  // hard dependencies, as in a minimum version of the "quasar"
  // package or a minimum version of "@quasar/app" CLI
  // api.compatibleWith('quasar', '^1.1.1')
  api.compatibleWith('@quasar/app', '^1.1.0 || ^2.0.0 || ^3.0.0-beta.1')

  // We extend /quasar.conf.js
  api.extendQuasarConf((conf) => {
    extendConf(api, conf)
  })
}
```

So you have api in your other hooks

Reply:
a bit more code on that usage:
```js
  // previous tp @quasar/app v2.0.0 the data had to be stringified
  for (const key in parsed) {
    target[key] = v1 === true ? JSON.stringify(parsed[key]) : parsed[key]
  }
```

Reply:
If the functions to the API are not the same as the Install api, we should create an issue ticket

Reply:
I made the issue